test/runtests.jl

```diff
@@ -235,7 +235,7 @@ using Test, InteractiveUtils
                 ==, !=, <, <=, >, >=,
                 +, -, *, /, ^, copysign, flipsign, max, min, rem)
             @test op(42, V4F64(v4f64)) === op(V4F64(42), V4F64(v4f64))
-            @test op(V4F64(v4f64), 42) === op(V4F64(v4f64), V4F64(42))
```

Comment:
It looks like `x^42` was too large for 32 bit machines? https://ci.appveyor.com/project/eschnett/simd-jl/builds/21409891

Reply:
This test fails for 64-bit machines and succeeds for 32-bit machines.

Reply:
Oops, I misread the table.  But still, 4^42 (note: `v4f64[end] == -4`) is much larger than `maxintfloat()` so I suppose decreasing the exponent makes sense?

But actually, I don't understand why it works in 64 bit Linux and not in 64 bit Windows.  Is it something we should worry about?  Is there any explanation?  Maybe Julia/LLVM emits different machine code just because they are on different machines (Travis log says ivybridge and Appveyor log says haswell)?